src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -222,7 +223,7 @@ class Wedge3D {
    */
```

Comment:
There is a remaining reference to the "spherical surface" at the start of the above documentation.